orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -901,42 +901,152 @@ const at::Tensor& resize_(
   return self;
 }
 
+/*
```

Comment:
I will delete all this commented out code. It is here to illustrate the generated code produced from the generator change in this PR.